app/services/container_dashboard_service.rb

```diff
@@ -233,7 +233,7 @@ def hourly_image_metrics
   def daily_image_metrics
     daily_image_metrics = Hash.new(0)
     daily_metrics.each do |m|
-      day = m.timestamp.strftime("%Y-%m-%d")
+      day = m.timestamp
```

Comment:
This looks problematic.

It gives me "Sat, 15 Jul 2017 22:29:00 UTC +00:00" .. which is not guaranteed to be parseable in javascript IIRC>